src/lib/core/a11y/focus-trap.spec.ts

```diff
@@ -167,8 +173,9 @@ class FocusTrapWithBindings {
   template: `
     <div cdkTrapFocus>
       <input>
-      <button id="last" cdk-focus-end></button>
-      <button id="first" cdk-focus-start>SAVE</button>
+      <button id="last" cdk-focus-region-end></button>
+      <button id="middle" cdk-focus-init></button>
+      <button id="first" cdk-focus-region-start>SAVE</button>
```

Comment:
The test was already like this, I think the reasoning is to make sure that it's really grabbing the element with the attribute, not just the first or last one. I changed it to something less confusing.